imports/ui/components/error.html

```diff
@@ -1,3 +1,91 @@
 <template name="error">
-    <img src="/icons/Error_graphic.png" >
-</template> 
+    <!-- Slide 1 Start-->
+    <section class="landing1">
+        <div class="landing1-box">
+            <div class="row landing1-top">
+                <div class="col-6">
+                    <div class="landing1-title">
+                        Online<br/> Geometry
+                        <br/> Viewer
+                    </div>
+                    <a href="/sign-up" class="button button-large button-red">Join</a>
+                    <a href="/log-in" class="button button-large button-blue">Login</a>
+                </div>
+                {{#if landingPageModel}}
+                <div class="col-6">
+                    <iframe width="350" height="250" src="/models/{{landingPageModel}}/shared=true" frameborder="0"></iframe>
+                </div>
+                {{/if}}
+            </div>
+            <div class="row landing1-points">
+                <div class="col-4 landing1-point">
+                    <div class="landing1-point-title">
+                        Simple
+                    </div>
+                    <div class="landing1-point-text">
+                        Just drag and drop your models and enjoy.
+                    </div>
+                </div>
+                <div class="col-4 landing1-point">
+                    <div class="landing1-point-title">
+                        Open
+                    </div>
+                    <div class="landing1-point-text">
+                        Built by volunteers around the world. Join us at <a href="https://github.com/BRLCAD/OGV-meteor">github</a>.
+                    </div>
+                </div>
+                <div class="col-4 landing1-point">
+                    <div class="landing1-point-title">
+                        Social
+                    </div>
+                    <div class="landing1-point-text">
+                        Embed and share your models anywhere on social sites, blogs or others.
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <div style="clear:both;"></div>
+    <!-- Slide 1 Over -->
+
+    <!-- Slide 2 Start-->
+    <section class="slide story" id="landing2" data-slide="2">
+        <div class="container">
+            <div class="row title-row">
+                <div class="col-12 landing2-title">This is what OGV does best</div>
+            </div>
+            <div class="row">
+                <div class="col-12 col-lg-3 col-sm-6">
+                    <p><img class="mobile-view-fileUpload" src="/icons/fileUpload.png" width="200px" height="200px" /></p>
```

Comment:
Is there a reason we are giving width height here, and not via styles?

Reply:
This is from the previous landingpage.html.

https://github.com/BRL-CAD/OGV-meteor/blob/development/imports/ui/pages/landingPage.html